docs/maintain-guides-how-to-nominate-kusama.md

```diff
@@ -35,3 +37,65 @@ Select them, confirm the transaction, and you're done - you are now nominating.
 ### Step 3: Stop nominating
 
 At some point, you might decide to stop nominating one or more validators. You can always change who you're nominating, but you cannot withdraw your tokens unless you unbond them. Detailed instructions are available [here](maintain-guides-how-to-unbond).
+
+# Using Command-Line Interface (CLI) 
+
+Apart from using the PolkadotJS UI to participate in the staking, you can do all these things in CLI instead. The CLI approach that allows you to interact with the Polkadot / Kusama network without going to the PolkadoJS dashboard.
+
+### Step 1: Install @polkadot/api-cli
+
+We assume you have installed [NodeJS with npm](https://nodejs.org). Run the following command to install the `@polkadot/api-cli` globally:
+
+```bash
+npm install -g @polkadot/api-cli
+```
+
+### Step 2. Bond your KSM
+
+Executing the following command:
+
+```bash
+polkadot-js-api --seed "MNEMONIC_PHRASE" tx.staking.bond CONTROLLER_ADDRESS NUMBER_OF_TOKENS REWARD_DESTINATION --ws WEBSOCKET_ENDPOINT
+```
+
+`CONTROLLER_ADDRESS`: An address you would like to bond to the stash account. (Stash and Controller can be the same address but it is not highly recommended due to the security reason)
```

Comment:
```suggestion
`CONTROLLER_ADDRESS`: An address you would like to bond to the stash account. Stash and Controller can be the same address but it is not recommended since it defeats the security of the two-account staking model.
```